package.json

```diff
@@ -33,5 +36,21 @@
   "engines": {
     "node": ">=10.15.0"
   },
+  "lint-staged": {
+    "*.js": [
+      "prettier --write --ignore-path .gitignore",
```

Comment:
I just noticed this is not scoped to the `marketplace` directory. IMHO we should keep it just to that directory for now since the other folders are not following prettier formatting right now.
(same goes for the line below)

Reply:
fixed